index.html

```diff
@@ -20,7 +20,7 @@
     title: "Tutorial"
     excerpt: "TermIt solves various problems, from creating vocabularies to annotating resources."
     url: "/tutorial"
-    btn_label: "Learn me how"
+    btn_label: "Show me how"
```

Comment:
I wanted to use different verb than in deployments, but as far as we're cancelling deployments page, i think its OK